README.md

```diff
@@ -14,18 +14,14 @@
 
 </div>
 
-This is not a browser-grade CSS parser. If you need one,
-use [cssparser](https://crates.io/crates/cssparser) +
-[selectors](https://crates.io/crates/selectors).
+This is not a browser-grade CSS parser. If you need one, use [cssparser](https://crates.io/crates/cssparser) + [selectors](https://crates.io/crates/selectors).
 
 Since it's very simple we will start with limitations:
 
 ## Limitations
 
-- [At-rules](https://www.w3.org/TR/CSS21/syndata.html#at-rules) are not supported.
-  They will be skipped during parsing.
-- Property values are not parsed.
-  In CSS like `* { width: 5px }` you will get a `width` property with a `5px` value as a string.
+- [At-rules](https://www.w3.org/TR/CSS21/syndata.html#at-rules) are not supported.  They will be skipped during parsing.
+- Property values are not parsed.  In CSS like `* { width: 5px }` you will get a `width` property with a `5px` value as a string.
```

Comment:
I think the previous version was correct. These are separate sentences.
```suggestion
- [At-rules](https://www.w3.org/TR/CSS21/syndata.html#at-rules) are not supported.
  They will be skipped during parsing.
- Property values are not parsed.
  In CSS like `* { width: 5px }` you will get a `width` property with a `5px` value as a string.
```

Reply:
Whoops, jet lag. Fixed.